src/components/StateChannelTransactionsPanel.vue

```diff
@@ -31,7 +32,7 @@ import { isDesktop } from '@/utils/screen'
 import PaginatedContent from '@/components/PaginatedContent'
 
 const stateChannelDetailsStore = useStateChannelDetailsStore()
-const { stateChannelTransactions } = storeToRefs(stateChannelDetailsStore)
+const { stateChannelTransactions, stateChannelTxCount } = storeToRefs(stateChannelDetailsStore)
```

Comment:
I do not really favor mixing `tx` and `transactions`. The logical question is whether it the same or a different kind of transaction. If it's the same, why is it called differently?

Reply:
That is a fair point, I copied Tx from some other function name but it looks like Transactions is the right chunk